<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgAssembly</class>
 <widget class="QDialog" name="CQDlgAssembly">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assembling Behaviour</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QToolButton" name="qqSetLocalMatrix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Set matrix</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="qqClose">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>110</y>
     <width>351</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqChildMatchValue">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Required match values for parent</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqParentMatchValue">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>191</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Required match values for child</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqHasMatrix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>361</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>When assembling, a specific local transform. matrix will be applied.</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqChildMatchValue</tabstop>
  <tabstop>qqParentMatchValue</tabstop>
  <tabstop>qqHasMatrix</tabstop>
  <tabstop>qqSetLocalMatrix</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
